src/core/chp.jl

```diff
@@ -159,6 +161,9 @@ function CHP(d::Dict;
     if !haskey(d, "fuel_cost_per_mmbtu")
         throw(@error("CHP must have the required fuel_cost_per_mmbtu input"))
     end
+
+    set_tech_sector_defaults!(d; sector=sector, federal_procurement_type=federal_procurement_type)
```

Comment:
Curious why this approach is used here but the `get_sector_defaults_techs` is used below to set the defaults in-line where they're defined?